### ISSUE TITLE
accessKeyId is not set

### ISSUE THREAD

**Issue by birben** [jsreport/jsreport-fs-store-aws-s3-persistence#2]
Hi,

I followed the exact following steps
```
npm install -g jsreport-cli
jsreport init
npm uninstall -g jsreport-cli
npm cache clean -f
npm install jsreport-fs-store
npm install jsreport-fs-store-aws-s3-persistence
```

After that, I changed the `jsreport.config.json` file as following
```
{
    "httpPort": 5488,
    "allowLocalFilesAccess": true,
    "store": {
        "provider": "fs"
    },
    "blobStorage": {
        "provider": "fs"
    },
    "templatingEngines": {
        "timeout": 10000
    },
    "chrome": {
        "timeout": 30000
    },
    "extensions": {
        "scripts": {
            "timeout": 30000
        },
        "fs-store": {
          "persistence": {
            "provider": "aws-s3"
          }
        },
        "fs-store-aws-s3-persistence": {
          "accessKeyId": "...",
          "secretAccessKey": "...",
          "bucket": "..."
        }
    }
}

``` 

and when I run 
```
node server.js
```

I got the following error
```
error: Error occured during reporter init Error: The fs store is configured to use aws s3 persistence but the accessKeyId is not set. Use store.persistence.accessKeyId or extensions.fs-store-aws-s3-persistence.accessKeyId to set the proper value.
    at module.exports (/Users/birben/tmp/reportjs/node_modules/jsreport-fs-store-aws-s3-persistence/lib/fileSystemS3.js:11:11)
    at reporter.fsStore.registerPersistence (/Users/birben/tmp/reportjs/node_modules/jsreport-fs-store-aws-s3-persistence/lib/main.js:7:8)
    at Object.load (/Users/birben/tmp/reportjs/node_modules/jsreport-fs-store/lib/provider.js:42:15)
    at Object.init (/Users/birben/tmp/reportjs/node_modules/jsreport-core/lib/store/documentStore.js:60:29)
    at Reporter.init (/Users/birben/tmp/reportjs/node_modules/jsreport-core/lib/reporter.js:188:32)
Error: The fs store is configured to use aws s3 persistence but the accessKeyId is not set. Use store.persistence.accessKeyId or extensions.fs-store-aws-s3-persistence.accessKeyId to set the proper value.
    at module.exports (/Users/birben/tmp/reportjs/node_modules/jsreport-fs-store-aws-s3-persistence/lib/fileSystemS3.js:11:11)
    at reporter.fsStore.registerPersistence (/Users/birben/tmp/reportjs/node_modules/jsreport-fs-store-aws-s3-persistence/lib/main.js:7:8)
    at Object.load (/Users/birben/tmp/reportjs/node_modules/jsreport-fs-store/lib/provider.js:42:15)
    at Object.init (/Users/birben/tmp/reportjs/node_modules/jsreport-core/lib/store/documentStore.js:60:29)
    at Reporter.init (/Users/birben/tmp/reportjs/node_modules/jsreport-core/lib/reporter.js:188:32)
```

When I comment out the following line in `main.js` line 11
```
definition.options = {}
```
it works fine.

Am I missing something or is this a bug?

Cheers

**Comment by pofider** [comment]
Ups, yes bug. You can use previous version 1.2.0.
However there is security consideration. The previous version exposes the aws access key through the API. 
I will release patch today.

Thanks for pointing it out.

**Comment by birben** [comment]
Thanks for the reply, looking forward to your patch :)

Cheers

**Comment by pofider** [comment]
Fixed in 1.2.2.
